File: 2085. Count Common Words With One Occurrence.ts
function countWords(words1: string[], words2: string[]): number {
    let map1 =new Map<string, number>(), map2=new Map<string, number>();
    for(let i:number=0;i<words1.length;i++) { map1.set(words1[i],map1.get(words1[i])+1||1) }
    for(let i:number=0;i<words2.length;i++) { map2.set(words2[i],map2.get(words2[i])+1||1) }
    let ans:number=0;
    for (const key of map1){ if (key[1]===1 && map2.get(key[0])===1)  ans++; }
    return ans;
};   

File: 2085. Count Common Words With One Occurrence.py
class Solution(object):
    def countWords(self, words1, words2):
        map1 = {}
        map2 = {}
        for word in words1:
            map1[word] = map1.get(word, 0) + 1
        for word in words2:
            map2[word] = map2.get(word, 0) + 1
        ans = 0
        for key, value in map1.items():
            if value == 1 and map2.get(key, 0) == 1:
                ans += 1
        return ans
        
